packages/web-vite/src/pages/HighlightsPage.tsx

```
@@ -0,0 +1,224 @@
+        <div className="spinner"></div>
+        <p>Loading your highlights...</p>
+      </div>
+    )
+  }
+
+  if (error) {
+    return (
+      <div className="error-state">
+        <h2>Something went wrong</h2>
+        <p>{error}</p>
+        <button onClick={() => window.location.reload()}>Try again</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)